setup.py

```diff
@@ -39,16 +38,21 @@ def get_version():
     ],
     license='MIT',
     install_requires=[
-        'django>=1.6',
+        'Django>=1.7',
+        'jsonfield',
+        'six',
```

Comment:
You can remove six, as it is vendored inside of Django https://github.com/django/django/blob/master/django/utils/six.py